ui/i18n/src/main/res/values/strings.xml

```diff
@@ -61,6 +59,10 @@
     <string name="home_continue_title">Continue listening</string>
     <string name="home_new_title">See what\'s new</string>
     <string name="home_downloads_title">Manage downloads</string>
+    <string name="home_surprise_empty_text">There is just one recent episode, not really much to be surprised here ;)</string>
+    <string name="home_continue_empty_text">You can add episodes here by downloading them or long pressing them and selecting "Add to queue".</string>
+    <string name="home_new_empty_text">New episodes will show up here. You can then decide whether they are interesting or not.</string>
+    <string name="home_downloads_empty_text">You can download episodes at each podcast page.</string>
```

Comment:
This text sounds a bit weird to me, to be honest. I would just write "download an episode to make it available offline" or something similar. You can basically download episodes from everywhere - saying that you need to go to a specific page could be confusing

Reply:
I agree, but that's what @keunes had in their mockup and I just translated it since, I assumed that what you wanted.
But I will adapt it...